checks/apps/quantumespresso/quantumespresso_check.py

```diff
@@ -1,136 +1,223 @@
-# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# Copyright 2016-2024 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
 # ReFrame Project Developers. See the top-level LICENSE file for details.
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
-import pathlib
-import re
-import sys
+import os
+import shutil
 
 import reframe as rfm
 import reframe.utility.sanity as sn
+import reframe.utility.udeps as udeps
+import uenv
 
-sys.path.append(str(pathlib.Path(__file__).parent.parent.parent / 'mixins'))
-from extra_launcher_options import ExtraLauncherOptionsMixin
-from sarus_extra_launcher_options import SarusExtraLauncherOptionsMixin
-from cuda_visible_devices_all import CudaVisibleDevicesAllMixin
-
-qe_tests = {
-    # R:total number of MPI ranks, T:threads per rank, P:walltime
-    'Au-surf': {
-        'zen2': {
-            # zen2 cpu nodes: 2sockets, 8 numa, 16c/numa = 128c (no MT)
-            'energy_reference': -11427.09017218,
-            'performance_reference': [{'R': 32, 'T': 4, 'P': 102.0}]
+
+qe_references = {
+    "Au surf": {
+        "gh200": {"time_run": (14.02, None, 0.05, "s")},
+        "zen2": {"time_run": (99.45, None, 0.05, "s")}, # 1m44s
+    },
+}
+
+
+slurm_config = {
+    "Au surf": {
+        "gh200": {
+            "nodes": 1,
+            "ntasks-per-node": 4,
```

Comment:
Is mps really needed with only 4 mpi per node ? or could we add a test with more than 4 mpi ?

Reply:
The wrapper is indeed not needed. QE seems to perform best with only 1 rank per gpu. 
Shall I replace it with `SLURM_GPUS_PER_TASK=1`?

Reply:
let's keep it as it is